### PR TITLE
(chore): add issues templates

### PR DESCRIPTION
**Pull request by FaGru3n**
### Description

enable default issue templates for project management

### Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up to date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] Copyright and license header are present on all affected files

### Additional information

- To request a review, check out [who's involved](https://projects.eclipse.org/projects/automotive.tractusx/who) to see a list of contributors and committers
- Use the [mailing list](https://accounts.eclipse.org/mailing-list/tractusx-dev) if you are unsure, who to contact, or if want to engage in a general discussion
- Check out our guide on [how to contribute](https://eclipse-tractusx.github.io/docs/oss/how-to-contribute)
- Check out our [Release Guidelines](https://eclipse-tractusx.github.io/docs/release)
- Check out the [Eclipse Foundation contribution guidelines](https://www.eclipse.org/projects/handbook/#contributing)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_template.md
Comment (carslen) [inline .github/ISSUE_TEMPLATE/bug_template.md:1]: What kind of bug shall be reported using this template? Product/component specific bugs? I'd expect these in the product/component related repository but not in a detatched repo.

Why do we use these front matter? It's super confiusing/cluttering the issue:
- Name + Label should be handled by label the issue
- title is in the title, why typing it again
- assignees?!

Or are these fields filled automatically by GH magic?

Remarks about the template itself:
- Keep it simple
- remove prefilled _go to_, _click on_, etc.
- trust in the abilitiy of people to describe the issue
- describe expected content with a comment

Comment (FaGru3n) [reply .github/ISSUE_TEMPLATE/bug_template.md:1]: @carslen  Thanks for your input. This is exactly the reason why i created this PR to get all information and thoughts form everyone as reviewers.

To your points using the template: 
> What kind of bug shall be reported using this template? 

This was not disscussed in workshop should be fine for both i guess, but is a first general approach to get the feedback and info for our managment plans, and what we want to share with.

> Name + Label should be handled by label the issue 

Name is the displayed name for the issue template and the label will be automatically added to the issue if template is used.

> title is in the title, why typing it again

Title is the field for a maybe prefixed Title for the issue that will be created with the Template

> assignees?!

Not clear at the moment, who should be resposible for the bugs, depends on the repo or problem where the template 
is implemented. But good for further discussions

> about is a useless phrase without added value?!

Here we can add more specific descriptions about the issue template.
image


to the Remarks: 

> remove prefilled go to, click on, etc.

The goal was the generate an issue template that is a bit alligned to the "state-of-the-art" issue templates in Open Source projects like GitHub itself describes with the default templates. But i share your opinion to minimize the requested information, makes it easier to fill out and less work for handling *bug* issues.

**Comment by SebastianBezold** [comment]
Hi all,

in my opinion, this is already too complicated, even if it is only 4 issue templates.
In my opinion, the only template needed is the `Feature` template. The reasons behind (my personal opinion)

- `Bugs` will be handled on the repositories. If contributors will find bugs, they will report it directly on the repository they are dealing with. Needing to know all the processes and boards is an unnecessary burden for contributors
- `Feature` and `Requirement` is the same to me. We should avoid naming things too closely related. This just makes it difficult in conversations.
- `Task` is again somewhat similar to `Feature` or `Requirement`. If this is meant to be a technical implementation step, then I would rather see this on the product repositories and let the maintainers decide on how to break features down to single task on their repositories

The GitHub project defined on `sig-project-managment` (-> typo in the name) should be used to plan features on a high level throughout the whole Tractus-X project. Its main purpose should be alignment. As soon, as we as a project are aligned on the next features, every team can decide, how to break things down to their repos. We can of course ask everyone to link the issues to the feature in `sig-project-managment'.

As of now, I also don't see the need of labels. `feature`, `requirement` or `task` is already covered. Feature is everything we have and other categories do not provide better understanding. We could keep `bug` and `documentation`. But `documentation` is an integral part of every feature, so I would always expect that. `bug`s could already be handled on the repo level. `enabler` is too unspecific to me. If it's a supportive task, then I would expect an issue on the involved repository, that links back to the overall feature issue.

My personal wish is, that we keep things simple and easy at the beginning. Start with a minimal set of options and only if we really find the need after the first couple of iterations, we can add more complexity.

What do you think?

**Review by SebastianBezold — DISMISSED** [review]
See my [last comment](https://github.com/eclipse-tractusx/sig-project-managment/pull/10#issuecomment-1628806009) on why I think we should not merge it like it is, but condense it to a simpler approach. 
Should we prepare some alternative proposals?

**Comment by SebastianBezold** [comment]
> @SebastianBezold i don't mind having more issue types as it makes the filtering and moving around easier / you will need mulitply tags anyway for later for org reasons.
> 
> I don't like the description of 'requirement' though as it doesn't reflect what the issue is for.
> 
> and the templates should be more specific to make more sense, otherwise the 'moving into the right category' can happen after the issue was created by some management person.

Ok, then I think we need better description on the differences between the templates. Anyone willing to propose a new feature should have it as easy as possible to choose the correct template. This only works, if the issue types are distinct enough. 
These differences could well be described in the `README.md` with hint on how the issues are treated differently

**Comment by FaGru3n** [comment]
> The GitHub project defined on `sig-project-managment` (-> typo in the name)

Thanks for pointing out:
was fixed in https://github.com/eclipse-tractusx/.eclipsefdn/pull/8

now availalbe under https://github.com/eclipse-tractusx/sig-project-management/

**Comment by SebastianBezold** [comment]
Hi all,
after discussions and pairing with @FaGru3n, I added an initial process description. In my opinion the current leaner approach with only feature and bug templates is a good start. This is easy to understand, also by contributors not yet too familiar with the Tractus-X ecosystem.
What do you think?

**Pull request merged**
